app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_, file_to_move, path_to_move = command.split()
+        if command_ == "mv":
+            place_to_move = path_to_move.split("/")[:-1]
+            for direction in place_to_move:
+                os.mkdir(direction)
+                os.chdir(f"{os.getcwd()}\\{direction}")
+            os.chdir(f"{'../' * len(place_to_move)}/")
```

Comment:
Do not use `/` in your path manipulations, relay on the `os` module to make the code platform agnostic.